pkgs/by-name/an/anyrun/package.nix

```diff
@@ -0,0 +1,67 @@
+{ lib
+, rustPlatform
+, fetchFromGitHub
+, pkg-config
+, wrapGAppsHook
+, atk
+, cairo
+, gdk-pixbuf
+, glib
+, gtk3
+, pango
+, stdenv
+, darwin
+, wayland
+, gtk-layer-shell
+}:
+
+rustPlatform.buildRustPackage rec {
+  pname = "anyrun";
+  version = "unstable-2023-10-10";
+
+  src = fetchFromGitHub {
+    owner = "kirottu";
+    repo = "anyrun";
+    rev = "f9073a6184f61a07bb584ecc47fbb8eb997e9216";
+    hash = "sha256-PuaKdM4Nems/wQ2B+K7hI2UJYld/+64NdkcsYD1YJwM=";
+  };
+
+  cargoHash = "sha256-npKM4Yk69Y00vZn88w96dZ4EohU+HAx328KcGRafD40=";
+
+  strictDeps = true;
+  enableParallelBuilding = true;
+  doCheck = true;
+
+  nativeBuildInputs = [
+    pkg-config
+    wrapGAppsHook
+  ];
+
+  buildInputs = [
+    atk
+    cairo
+    gdk-pixbuf
+    glib
+    gtk3
+    gtk-layer-shell
+    pango
+  ] ++ lib.optionals stdenv.isDarwin [
+    darwin.apple_sdk.frameworks.Security
+  ] ++ lib.optionals stdenv.isLinux [
+    wayland
+  ];
+
+  preFixup = ''
```

Comment:
I'd rather if we built the plugins on demand, with something like `withPlugins ? false` that the user can override if they would like the plugins.

Reply:
It seems that building without plugins breaks the program, unfortunately